nmdc_schema/create_legacy_orphan_deletion_requests.py

```diff
@@ -0,0 +1,33 @@
+import json
+import csv
+
+
+def gen_delete(delete_list: list, key: str, limit: int, del_coll: str):
+    if limit not in (0, 1):
+        raise ValueError("limit must be 0 or 1.")
+    delete_statement = []
+    for record in delete_list:
+        delete_statement.append({"q": {key: record}, "limit": limit})
+    request_body_json = {"delete": del_coll, "deletes": delete_statement}
+    request_body_file = del_coll + "_request_body.json"
+    with open(request_body_file, "w") as f:
+        json.dump(request_body_json, f)
+
+
+# delete legacy orphan functional_annotation_agg records
+delete_agg_list = []
+with open("to_delete_nmdc.functional_annotation_agg.csv", newline="") as csvfile:
```

Comment:
Long term, I'd recommend this file path be a CLI argument.

Short term, I'd recommend it be a variable defined at the top of the file.

Immediate term, I'm OK with this being merged in as is.